src/ui_utils.c

```diff
@@ -533,9 +541,13 @@ void ui_update_menu_copy_items(GeanyDocument *doc)
 		enable = gtk_text_buffer_get_selection_bounds(buffer, NULL, NULL);
 	}
 
-	len = G_N_ELEMENTS(widgets.menu_copy_items);
-	for (i = 0; i < len; i++)
-		ui_widget_set_sensitive(widgets.menu_copy_items[i], enable);
+	set_menu_copy_items_sensitive(enable);
+}
+
+
+void ui_restore_menu_copy_items()
```

Comment:
This function indirection doesn't add clarity to me, so I'd rather expose the `set_sensitive()` part directly.